cmd/osm-crd-converter/osm-crd-converter.go

```diff
@@ -0,0 +1,193 @@
+// Package main implements the main entrypoint for osm-crd-converter and utility routines to
+// bootstrap the various internal components of osm-crd-converter.
+// osm-crd-converter provides crd conversion capability in OSM.
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/pflag"
+	admissionv1 "k8s.io/api/admissionregistration/v1"
+	corev1 "k8s.io/api/core/v1"
+	apiclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/typed/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```

Comment:
🤔  I would leave a comment explaining why this is needed.

Reply:
good catch, I have no idea why that package got added. Its not needed so I've removed it